### PR TITLE
[docs] V11 UI shell color updates

### PR DESCRIPTION
**Pull request by aagonzales**
This PR updates the color tokens on the Style tab for:
- UI shell header
- UI shell left panel
- UI shell right panel

### CODE REVIEW

File: src/pages/components/button/usage.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Button
+title: BUTTONS
```
Comment (alisonjoseph) [inline src/pages/components/button/usage.mdx:2]:
```suggestion
title: Button
```
😆

File: src/pages/components/accordion/usage.mdx [hunk 2]
```diff
@@ -8,7 +8,7 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 
 <PageDescription>
 
-An accordion is a vertically stacked list of headers that reveal or hide
+THE accordion is a vertically stacked list of headers that reveal or hide
```
Comment (alisonjoseph) [inline src/pages/components/accordion/usage.mdx:11]:
```suggestion
The accordion is a vertically stacked list of headers that reveal or hide
```

**Review by andreancardona — APPROVED** [review]
looks. good!

**Review by alisonjoseph — APPROVED** [review]
🚀

**Pull request merged**